src/components/pages/AdminCredential/AdminCredentialElements.tsx

```diff
@@ -303,7 +303,7 @@ export default function AdminCredentialElements() {
       flex: 1,
       renderCell: ({ row }: { row: CredentialData }) => (
         <>
-          {UserService.hasRole(ROLES.REVOKE_CREDENTIALS_ISSUER) &&
+          {userHasPortalRole(ROLES.REVOKE_CREDENTIALS_ISSUER) &&
```

Comment:
@oyo this role is the role of the issuer component
<img width="286" alt="image" src="https://github.com/eclipse-tractusx/portal-frontend/assets/94133633/c626b306-9679-464d-b9ee-20b296d9b3b8">